package.json

```diff
@@ -23,9 +23,32 @@
       {
         "command": "be-human.trackTime",
         "title": "Time Tracking"
+      },
+      {
+        "command": "be-human.resetTime",
+        "title": "Reset Time Tracking"
+      },
+      {
+        "command": "be-human.pauseStart",
+        "title": "Pause Time Tracking"
+      },
+      {
+        "command": "be-human.pauseStop",
+        "title": "Resume Time Tracking"
+      }
```

Comment:
Could these be `be-human.pause` and `be-human.resume`? "pauseStop" is a little confusing.

Reply:
The pause functions are not active, only the reset.